OpenOversight/app/api/v1.py

```diff
@@ -0,0 +1,56 @@
+from http import HTTPMethod
+
+from flask import Blueprint, Response, jsonify
+from sqlalchemy.orm import Query
+
+from OpenOversight.app.models.database import Department
+from OpenOversight.app.utils.flask import limiter
+
+
+v1 = Blueprint("v1", __name__, url_prefix="/api/v1")
+
+
+def objs_to_dicts_jsonify(obj_list: Query) -> Response:
+    return jsonify([o.to_dict() for o in obj_list])
+
+
+@v1.route("/departments/<int:department_id>/assignments", methods=[HTTPMethod.GET])
+@limiter.limit("5/minute")
+def get_dept_assignments(department_id: int) -> Response:
+    assignments = Department.get_assignments(department_id)
+    return objs_to_dicts_jsonify(assignments)
```

Comment:
I know we already vend all our data for CSV exports but I think we may want to consider adding pagination and filtering to reduce the amount of data we vend out per request.